R/setConfig.R

```diff
@@ -58,26 +52,29 @@
 #' "bzip2" or "xz" specify the type of compression.
 #' @param hash specifies the used hashing algorithm. Default is "xxhash32" and
 #' all algorithms supported by \code{\link[digest]{digest}} can be used.
-#' @param delete_cache Boolean deciding whether a temporary cache folder (as
-#' created by retrieveInput) should be deleted after completion or not.
-#' @param diagnostics file name for additional diagnostics information (without file ending).
+#' @param diagnostics Either FALSE (default) to avoid the creation of additional
+#' log files or a file name for additional diagnostics information (without file ending).
 #' 2 log files are written if a file name is provided (a compact version with the most
 #' relevant information and a full version with all available details).
-#' @param nocores integer number of cores to use
 #' @param debug Boolean which activates a debug mode. In debug mode all calculations will
 #' be executed with try=TRUE so that calculations do not stop even if the previous calculation failed.
 #' This can be helpful to get a full picture of errors rather than only seeing the first one. In addition
 #' debug=TRUE will add the suffix "debug" to the files created to avoid there use in productive runs.
 #' Furthermore, with debug=TRUE calculations will be rerun even if a corresponding tgz file
 #' already exists.
-#' @param indentationCharacter character used for indenting the output of nested function calls
 #' @param maxLengthLogMessage in log messages evaluated arguments are printed if the resulting message
 #' is shorter than this value, otherwise arguments are shown as passed, potentially  with unevaluated variable names
 #' @param .cfgchecks boolean deciding whether the given inputs to setConfig should be checked for
 #' consistency or just be accepted (latter is only necessary in very rare cases and should not be used
 #' in regular cases)
 #' @param .verbose boolean deciding whether status information/updates should be shown or not
 #' @param .local boolean deciding whether options are only changed until the end of the current function execution
+#' OR environment for which the options should get changed.
```

Comment:
As suggested above (use `evalq` with the correct `envir`), maybe we don't need this extra flexibility/complexity here.

Reply:
what are you referring here to?

Reply:
To my last comment on R/addMapping.R